packages/lit-element/src/lib/css-tag.ts

```diff
@@ -31,24 +29,22 @@ export class CSSResult {
         'CSSResult is not constructable. Use `unsafeCSS` or `css` instead.'
       );
     }
-
     this.cssText = cssText;
   }
 
   // Note, this is a getter so that it's lazy. In practice, this means
   // stylesheets are not created until the first element instance is made.
   get styleSheet(): CSSStyleSheet | null {
-    if (this._styleSheet === undefined) {
-      // Note, if `supportsAdoptingStyleSheets` is true then we assume
-      // CSSStyleSheet is constructable.
-      if (supportsAdoptingStyleSheets) {
-        this._styleSheet = new CSSStyleSheet();
-        this._styleSheet.replaceSync(this.cssText);
-      } else {
-        this._styleSheet = null;
-      }
+    let sheet: CSSStyleSheet | null;
+    // Note, if `supportsAdoptingStyleSheets` is true then we assume
+    // CSSStyleSheet is constructable.
+    if (supportsAdoptingStyleSheets) {
+      sheet = new CSSStyleSheet();
+      sheet.replaceSync(this.cssText);
+    } else {
+      sheet = null;
     }
-    return this._styleSheet;
+    return sheet;
```

Comment:
If the return type changes to `undefined`, then you can just handle the `supportsAdoptingStyleSheets` case and save some bytes

```ts
if (supportsAdoptingStyleSheets) {
  const sheet = new CSSStyleSheet();
  sheet.replaceSync(this.cssText);
  return sheet;
}
```

Reply:
Done.